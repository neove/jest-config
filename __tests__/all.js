test('生成snapshot', () => {
    const util = require('../index.js');
    let conf = util();
    expect(conf).toHaveProperty('transform');
    expect(conf).toHaveProperty('moduleDirectories');
    expect(conf).toHaveProperty('globals');
    expect(conf).toHaveProperty('collectCoverageFrom');
    expect(conf).toHaveProperty('moduleNameMapper');
    expect(conf).toHaveProperty('testRegex');
    expect(conf).toHaveProperty('moduleFileExtensions');
})