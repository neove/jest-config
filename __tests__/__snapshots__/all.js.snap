// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`生成snapshot 1`] = `
Object {
  "collectCoverageFrom": Array [
    "**/src/**/*.{js,jsx}",
    "!**/node_modules/**",
  ],
  "globals": Object {},
  "moduleDirectories": Array [
    "/Users/neo/Projects/talent-ui-2.0/Components/jest-config/src",
    "node_modules",
  ],
  "moduleFileExtensions": Array [
    "ts",
    "tsx",
    "js",
    "jsx",
    "json",
  ],
  "moduleNameMapper": Object {
    "&(.*)$": "<rootDir>/src$1",
    "\\\\.(css|less|scss)$": "/Users/neo/Projects/talent-ui-2.0/Components/jest-config/utils/style-mock.js",
    "\\\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "/Users/neo/Projects/talent-ui-2.0/Components/jest-config/utils/file-mock.js",
  },
  "testRegex": "(/__tests__/.*|(\\\\.|/)(test|spec))\\\\.(jsx?|tsx?)$",
  "transform": Object {
    "^.+\\\\.jsx?$": "/Users/neo/Projects/talent-ui-2.0/Components/jest-config/utils/js-processor.js",
    "^.+\\\\.tsx?$": "/Users/neo/Projects/talent-ui-2.0/Components/jest-config/utils/ts-processor.js",
  },
}
`;
